zilliqa/src/message.rs

```diff
@@ -170,10 +189,11 @@ pub struct BlockBatchResponse {
 pub struct IntershardCall {
     pub source_address: H160,
     pub target_address: Option<H160>,
+    pub source_chain_id: u64,
+    pub bridge_nonce: u64,
```

Comment:
To avoid confusion between UCCB and x-bridge, which is a specific application built on UCCB, we've renamed UCCB to Universal Cross-Chain Broker in the whitepaper - up to you if you want to rename `bridge_nonce` to `uccb_nonce`...